main.js

```diff
@@ -92,7 +92,7 @@ const Expander = function (el, opts) {
 	}
 
 	if (typeof this.opts.shrinkTo === 'number' && !this.opts.countSelector) {
-		throw('when collapsing to a number of items specify a selector to identify how many items exist');
+		throw new Error('when collapsing to a number of items specify a selector to identify how many items exist');
```

Comment:
We used to throw a string here, I've changed to throwing an error, reckon this is a breaking change?

Reply:
Agh I'm not sure.. _maybe_, depends if anyone's relying on it being a string, which it's nearly impossible for us to tell